nir/src/main/scala/scala/scalanative/nir/package.scala

```diff
@@ -1,13 +1,27 @@
 package scala.scalanative
 
 package object nir {
+
   type LocalName = String
```

Comment:
I couldn't document this alias because I don't know what a "local" is.

Reply:
Here the `LocalName` is the name of variable in the source code. The `Local` is the SSA identifier in the scope of given `Defn.Define`, so it can be either a function parameter or an identifier of an instruction. 
To preserve the original name of the variable during transformations of the code, and make it available in the backend for generating debug metadata, each name of variable is paired with the `Local` for the result of mentioned variable 

For example `val x = 2; val y = x + 2`, might be represented as folows, where the `%` is the SSA identifier`, and `<...> is the local name of given identifier
```
 %1 <x> = 2
 %2 <y> = add(%1, 2)
```

Reply:
Oh wow, I'm surprised that there can be a correspondence so direct between Scala and NIR.